CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ Changes since v2.20
 ### Additions
 - "More info" support for DOI style resources (when `:enable-doi` flag is set) (#2701)
 
+### Fixes
+- Reviewers are no longer able to see private field questions in generated application pdf. Forms are also no longer rendered in application page if they contain only private fields (#2161)
```

Comment:
the feature feels like a fix, although i labeled my commits as `feat: ...`. i'm happy changing the format either way though :)

Reply:
We do not have such rigid naming policy, so your commits are absolutely fine!

Reply:
It's simply a design choice that was changed, and code updated to match. Feat is fine.

Reply:
fixed